app/assets/javascripts/angular/controllers/pages/create_recipe_controller.js.erb

```diff
@@ -48,36 +75,84 @@ app.controller('createRecipeCtrl', ['$scope', '$http', '$routeParams', function(
     }
 
     $(function() {
-      $( "#slider" ).slider({
-        // range: true,
+      $( "#prep-time-slider" ).slider({
         min: 0,
         max: 1440,
-        values: [ 0, 1440 ],
+        values: [0],
         slide: function( event, ui ) {
+          $( "#prep-time-amount" ).val( timeConversion(ui.values));
+          $scope.create_parameters.timeframe.prepTime = ui.values;
+          console.log($scope.create_parameters);
+        }
+      });
+      // initial slider values
+      $( "#prep-time-amount" ).val( timeConversion(0));
+    });
 
-          // $( "#amount" ).val( timeConversion(ui.values[0]) + " to " + timeConversion(ui.values[ 1 ]) );
+    $(function() {
+      $( "#cook-time-slider" ).slider({
+        min: 0,
+        max: 1440,
+        values: [0],
+        slide: function( event, ui ) {
+          $( "#cook-time-amount" ).val( timeConversion(ui.values));
+          $scope.create_parameters.timeframe.cookTime = ui.values;
+          console.log($scope.create_parameters);
+        }
+      });
+      // initial slider values
+      $( "#cook-time-amount" ).val( timeConversion(0));
+    });
 
-          $scope.search_parameters.timeframe = ui.values;
-          console.log($scope.search_parameters);
+    $(function() {
```

Comment:
Should turn slider into a directive when you have time so you don't need to repeat code